### PR TITLE
Repost of previous pull request

### PR DESCRIPTION
**Pull request by fuyuton**
This pull request contains the following:
Are you doing well? (I'm sorry I'm not used to it)

Added to use SNI
close?の追加
add reference variable
Update websocket-client-simple.gemspec
fix SSLError

### CODE REVIEW

File: lib/websocket-client-simple/version.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 module WebSocket
   module Client
     module Simple
-      VERSION = "0.3.0"
+      VERSION = "0.3.3"
```
Comment (unasuke) [inline lib/websocket-client-simple/version.rb:4]: Don't edit the version, please. The version is changed at release by the maintainer.

File: websocket-client-simple.gemspec [hunk 2]
```diff
@@ -18,11 +18,11 @@ Gem::Specification.new do |spec|
...
-  spec.add_development_dependency "eventmachine"
+  spec.add_development_dependency "rake", "~> 12.3.2"
+  spec.add_development_dependency "minitest", "~> 5.11.3"
+  spec.add_development_dependency "websocket-eventmachine-server", ">= 0"
+  spec.add_development_dependency "eventmachine", "~> 1.2.7"
 
-  spec.add_dependency "websocket"
-  spec.add_dependency "event_emitter"
+  spec.add_dependency "websocket", "~> 1.2.8"
+  spec.add_dependency "event_emitter", "~> 0.2.6"
```
Comment (unasuke) [inline websocket-client-simple.gemspec:27]: Why did you edit those lines? It does not affect actually and not breaking, also not needed.

**Review by unasuke — CHANGES_REQUESTED** [review]
Thank you for your pull request! 😄  And I have a favor to ask of you.

You wrote that in the description. 

> Added to use SNI
> close?の追加
> add reference variable
> Update websocket-client-simple.gemspec
> fix SSLError


It looks like doing some things in one pull request.
I want you to create one pull request for one purpose.

**Comment by fuyuton** [comment]
got it.

**Pull request closed without merging**